junit5/README.md

```diff
@@ -55,37 +62,39 @@ While both are ultimately achieving the same thing, each opts for a different ap
-In fact, you don't need anything except the JUnit `@ExtendWith` and our extension will try its best to find out what classes should be added to Weld container as beans.
-This of course makes some assumptions on your tests which may not always be met, hence there is bunch of annotations which allow you to configure the container.
-Pros of this approach are quick setup for basic cases, less verbose code and that eerie feeling that things are happening automagically.
-On the not so bright side, automatic config is not almighty and in some cases will falter forcing you to add some configuration via annotations.
-Last but not least, overly complex test scenarios may mean loads of annotations and you may be better off with the former extension.
+In fact, you don't need anything except the JUnit `@ExtendWith` and our extension will try its best to find out what classes should be added to the Weld container as beans.
+This of course makes some assumptions on your tests which may not always be met, hence there is a bunch of annotations which allow you to configure the container further.
+Advantages of this approach are quick setup for basic cases, less verbose code and that eerie feeling that things are happening automagically.
+On the not so bright side, automatic config is not almighty and in some cases will falter, forcing you to add some configuration via annotations.
+Last but not least, overly complex test scenarios may mean loads of annotations, and you may be better off with the former extension.
 
 No matter what extension you choose, do not mix them together!
 
 ## WeldJunit5Extension
 
 The simplest way to use this extension is to annotate your test class with `@ExtendWith(WeldJunit5Extension.class)`.
 If you are in for shorter annotations, you can also use `@EnableWeld`.
-With just these annotations, Weld container will be started before each test is run and stopped afterwards.
-This default behaviour includes:
+With just these annotations, the Weld container will be started before each test is run and stopped afterwards.
 
-* Bootstrapping Weld SE container with
+This default behaviour includes:
+* Bootstrap Weld SE container with
   * Disabled discovery
-  * Added test class package as source of beans
   * Disabled concurrent deployment
-* Injecting into test instance, e.g. into all `@Inject` fields
-* Injecting into method parameters of your test methods
-  * In case the type of the parameter matches a known and resolvable bean
-  * By default, Weld is greedy and will try to resolve all parameters which are known as bean types in CDI container
-  * If you wish to change this behaviour, please refer to [additional configuration section](#explicit-parameter-injection)
-* Shutting down the container after test is done
-
-And this is how you achieve it:
+  * Added test class package as source of beans
+* All discovered `WeldJunitEnricher`s are called to further configure the test environment
```

Comment:
It was second because it was in execution order.
I just thought it maybe was forgotton to add to documentation like `@ExcludeBean` in the JavaDoc for example.
I don't like hidden features. :-D
If features are for specific edge cases, I prefer to have it documented explicitly for which edge cases it is necessary.
But ok, I removed it again.

Reply:
To be precise, it shouldn't be a hidden feature and we should have it documented somewhere (which we should fix in some other PR).
I just don't think it needs to get this level of visibility because vast majority of users don't even need to know it exists, much less what it does.